packages/xo-server-perf-alert/src/index.js

```diff
@@ -157,6 +157,12 @@ export const configurationSchema = {
       items: {
         type: 'object',
         properties: {
+          all: {
```

Comment:
How will that evolve in the feature?

What will happen to this setting if/when we had more smart mode configuration?

Reply:
- Leave it
- Manage what is added 
- add a description in front of what we add as new setting
or
 - Plugin takes into account the smart schema (as we do in the backups)

Reply:
IMHO, for smooth transition to a full smart mode, the setting should be called `smartMode`, and when it will be implemented, it should match all objects by default.